README.md

```diff
@@ -1,5 +1,10 @@
 # SpringBoard - Intermediate DataScience with Python
-This repo contains Projects done as part of Springboard's Intermediate Data science with Python.
 
-#### data_wrangling_json - Hands on with getting meaningful data from JSON format files.
-#### Capstone_Project - Contains Jupyter notebooks(Code & reports) and data relevant to the capstone project
+## Inroduction
+This repository contains all Projects and excersises done as part of Springboard's Intermediate Data science with Python.
+
+## Contents
+
+1.  [Course_Work_Excersises - This folder contains all the projects and excersises done as part springboard's coursework](https://github.com/nishalpattan/SPRINGBOARD/tree/repo_clean/Course_Work_Excersises)
```

Comment:
@nishalpattan Good stuff in adding the link. I, however, would suggest to limit the link only to the _Course_Work_Excercises_ part